### ISSUE TITLE
Business Objects from Different End Points

### ISSUE THREAD

**Issue by chatzikk** [MarimerLLC/cslaforum#371]
I have a three tier WinForm application and the data portal is hosted on an IIS. 
Now I need to retrieve some of the business objects of the application from a data portal on a different IIS. Is it possible to configure the application to use two different end points?

**Comment by rockfordlhotka** [comment]
Yes. To do this you need to create a custom `Csla.DataPortalClient.IDataPortalProxy` that in turn creates the appropriate data portal proxy with the type and configuration you need to talk to each of your data portal servers.

The types you need are in the `Csla.DataPortalClient` namespace and you can see the code here: https://github.com/MarimerLLC/csla/tree/master/Source/Csla.Shared/DataPortalClient
